docs/usage.md

```diff
@@ -126,7 +61,9 @@ First, go to the [nf-core/mhcquant releases page](https://github.com/nf-core/mhc
 
 This version number will be logged in reports when you run the pipeline, so that you'll know what you used when you look back in the future.
 
-## Main arguments
+## Core Nextflow arguments
+
+> **NB:** These options are part of Nextflow and use a _single_ hyphen (pipeline parameters use a double-hyphen).
 
 ### `--input`
```

Comment:
As far as I know the description of these parameters should also be deleted from here.

Reply:
yes, this should be removed now, as the description is already included in the parameter json file